### ISSUE TITLE
Mobile - Need to tap 2 times to open node of a site

### ISSUE THREAD

**Issue by Julien-Dubois-eXo** [Meeds-io/meeds#1459]
**Steps to reproduce**

1. Open in mobile an app
2. Tap on the navigation menu
3. Tap on a site with sub-node (Contribue or My Craft)
4. Tap on a sub node (Program, Perks, Task etc)
5. See error

https://github.com/Meeds-io/meeds/assets/105866381/c3a62c20-85f5-4945-bacf-c2393b7420cd

**(optional) Affected version**

Indicate the version where the bug is observed.

**Current behaviour**

On the first tap nothing happens, the page is not opened. Tap a second time the page opens.

**Expected behavior**

The page is opened on the first tap.

**(optional) Affected environments**  
Please complete the following information:

**Comment by srenault-meeds** [comment]
Indeed, and I have noticed this is the same when using desktop (especially when using admin site)
@margondicco can you confirm the expected please?
